### PR TITLE
[MM-54794] Convert `./components/widgets/icons/fa_next_icon.tsx` to function component

### PR DESCRIPTION
**Pull request by fyfirman**
#### Summary
Convert `./components/widgets/icons/fa_next_icon.tsx` to function component. I also update the snapshot since it has different result.

#### Ticket Link
Fixes https://github.com/mattermost/mattermost/issues/24758
Jira [MM-54794](https://mattermost.atlassian.net/browse/MM-54794)

#### Screenshots
image
Example place in Emoji List

#### Release Note

```release-note
NONE
```

[MM-54794]: https://mattermost.atlassian.net/browse/MM-54794?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webapp/channels/src/components/widgets/icons/fa_next_icon.tsx [hunk 1]
```diff
@@ -8,21 +8,17 @@ import LocalizedIcon from 'components/localized_icon';
...
-    public render(): JSX.Element {
-        const className = 'icon icon-chevron-right' + (this.props.additionalClassName ? ' ' + this.props.additionalClassName : '');
-        return (
-            <LocalizedIcon
-                className={className}
-                title={{id: t('generic_icons.next'), defaultMessage: 'Next Icon'}}
-            />
-        );
-    }
+function NextIcon(props: Props) {
```
Comment (larkox) [inline webapp/channels/src/components/widgets/icons/fa_next_icon.tsx:14]: Again, the same changes as in https://github.com/mattermost/mattermost/pull/24787

**Review by larkox — CHANGES_REQUESTED** [review]
Great again. And again the same three changes :sweat_smile:

**Comment by fyfirman** [comment]
I just fixed it @larkox , thanks for the feedback.

**Review by larkox — APPROVED** [review]

**Review by devinbinnie — APPROVED** [review]
LGTM! Thanks @fyfirman!

**Comment by varghesejose2020** [comment]
@lindy65 I would like to test this PR as a QA.

**Comment by lindy65** [comment]
Thanks @varghesejose2020 🎉 

Log into the test server above using the sysadmin credentials

image

- Go to the system console from Main Menu (grid icon) > Authentication > AD/LDAP
- Change `Synchronization Interval (minutes):` setting and check that it Saves without error
- Then click on `AD/LDAP Synchronize Now` and check that the process completes without error

**Comment by lindy65** [comment]
Thanks @varghesejose2020 🎉 

- Log into the test server above using the sysadmin credentials
- From the Main Menu (grid icon) select Integrations
- Then select Custom Emoji
- Create a custom emoji and check that it saves correctly
- Go back to channels and make a post using the emoji you created

**Comment by varghesejose2020** [comment]
I tested the changes on the Test Server and it LGTM.
image
image
created a custom emoji and it posted on channel and it working as expected

**Review by lindy65 — APPROVED** [review]
Thanks @fyfirman - looks good to merge 👍 

Thanks for testing @varghesejose2020 🎉

**Comment by larkox** [comment]
Thanks for the contribution!

**Pull request merged**